docs/guides/cache-with-platformatic-watt.md

```diff
@@ -1,150 +1,332 @@
-# How to setup HTTP caching with Platformatic Watt
+# How to Setup HTTP Caching with Watt
 
-This guide will help you set up HTTP caching in your Node.js application using Platformatic Watt, the Node.js Application Server.
+## Problem
 
-## Prerequisites 
+You need to improve your application's performance by caching HTTP responses, but you want:
+- Intelligent cache invalidation when data changes
+- Fine-grained control over what gets cached and for how long
+- Ability to invalidate cache by specific tags or routes
+- Built-in caching without external dependencies like Redis
 
-Before starting, ensure you have the following installed:
+**When to use this solution:**
+- Your API serves data that doesn't change frequently
```

Comment:
Define frequently, anything that can work with potentially "stale" data for up to 1s is ok.